examples/advanced_circuits_algorithms/Randomness/Randomness_Generation.ipynb

```diff
@@ -82,10 +81,6 @@
     "# magic word for producing visualizations in notebook\n",
     "%matplotlib inline\n",
     "\n",
-    "# install a pip package in the current Jupyter kernel\n",
-    "#import sys\n",
-    "#!{sys.executable} -m pip install cvxpy\n",
-    "\n",
     "# import convex solver\n",
     "import cvxpy as cp"
    ]
```

Comment:
I think the previous output object (name: stdout, output_type: stream) containing the text "Requirement already satisfied ... should no longer be needed.

Reply:
Removed, Thanks!